tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Logging/Log4Net/ILoggingEvent.cs

```diff
@@ -0,0 +1,23 @@
+// <copyright file="ILoggingEvent.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections;
+using System.ComponentModel;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Log4Net
+{
+    /// <summary>
+    /// GraphQL.Language.AST.Document interface for ducktyping
```

Comment:
Copy/paste? 😄

Reply:
Yep! 😆 Thanks for spotting, resolved in the latest commits